src/test/java/tests/LoginTest.java

```diff
@@ -77,4 +77,23 @@ public void tc04_UserIsRedirectedToBookPageAfterLoggingIn() {
         LogHelper.info("Verify that user is redirected to Book Ticket page after logging in");
         Assert.assertEquals(actual, "Book ticket", "Navigate incorrectly to Book ticket page");
     }
+
+    @Test(description = "System shows message when user enters wrong password several times")
+    public void tc05_SystemShowsMessageWhenUserEntersWrongPasswordSeveralTimes() {
+        LogHelper.info("Click login tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Enter valid data into Username textbox except Password textbox and click login button. Repeat 3 times");
+        for (int i = 0; i < 4; i++) {
```

Comment:
wrap this in a method

Reply:
fixed

Reply:
what did you fix @taile47711 ?

Reply:
Sorry about that, I pushed wrong git. Please check again